.github/workflows/release-baml-language-alpha.yml

```
@@ -184,21 +184,50 @@ jobs:
+            os: blacksmith-4vcpu-ubuntu-2204
+            binary: baml-cli
+            pack_host: baml-pack-host
+            archive_ext: tar.gz
+            cross: true
           - target: x86_64-pc-windows-msvc
             os: blacksmith-4vcpu-windows-2025
             binary: baml-cli.exe
             pack_host: baml-pack-host.exe
             archive_ext: zip
+          - target: aarch64-pc-windows-msvc
+            os: blacksmith-4vcpu-windows-2025
```

actionlint (1.7.12)

[error] label "blacksmith-4vcpu-windows-2025" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)